app/assets/stylesheets/application/modules/_exercise_assignment.scss

```diff
@@ -6,7 +6,28 @@
   margin: 10px 0;
 }
 
-.editor-overlap {
+.mu-editor-overlap {
   z-index: 0;
   position: relative;
 }
+
+.mu-editor {
+  .form-group {
+    margin: 10px 0;
+  }
+  input[type=radio] {
```

Comment:
Yes, I think so. However I prefer to first merge this PR and later improve its styling. Would you want to open an issue and submit proposal about how it should look?